.github/workflows/publish-command.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           GCP_GSM_CREDENTIALS: ${{ secrets.GCP_GSM_CREDENTIALS }}
       - run: |
           echo "$SPEC_CACHE_SERVICE_ACCOUNT_KEY" > spec_cache_key_file.json && docker login -u airbytebot -p ${DOCKER_PASSWORD}
-          ./tools/integrations/manage.sh publish airbyte-integrations/${{ github.event.inputs.connector }} ${{ github.event.inputs.run-tests }} --publish_spec_to_cache
+          ./tools/integrations/manage.sh publish airbyte-integrations/${{ github.event.inputs.connector }} false --publish_spec_to_cache
```

Comment:
make sure not to merge this to master. FYI you can always toggle this option via `/publish ... run-tests=false`

Reply:
yes of course I will revert 
thanks for `run-tests=false`